test/Microsoft.DotNet.ProjectModel.Tests/PackageDependencyProviderTests.cs

```diff
@@ -65,5 +65,18 @@ public void TestDuplicateDefaultDesktopReferences(string sampleName, bool withLo
 
             Assert.Equal(4, context.RootProject.Dependencies.Count());
         }
+        
+        [Fact]
+        public void NoDuplicateReferencesWhenFrameworkMissing()
+        {
+            var instance = TestAssetsManager.CreateTestInstance("TestMicrosoftCSharpReferenceMissingFramework")
+                                            .WithLockFiles();
+
+            var context = new ProjectContextBuilder().WithProjectDirectory(instance.TestRoot)
+                                                     .WithTargetFramework("net99")
+                                                     .Build();
+
+            context.LibraryManager.GetLibraries().ToDictionary(l => l.Identity.Name);
```

Comment:
Seems xunit removed `Assert.DoesNotThrow`, which doesn't do anything but it looks nice